server/src/main/java/org/candlepin/cache/JCacheManagerProvider.java

```diff
@@ -16,17 +16,26 @@
 
 import com.google.inject.Provider;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URISyntaxException;
+
+import javax.cache.CacheException;
 import javax.cache.CacheManager;
 import javax.cache.Caching;
 import javax.cache.spi.CachingProvider;
 
 /**
  * Provides object cache by indexed by String.
- * @author fnguyen
  *
+ * @author fnguyen
  */
 public class JCacheManagerProvider implements Provider<CacheManager> {
 
+    private static final String CACHE_MANAGER_URI = "ehcache.xml";
```

Comment:
If we're going to move the configuration file reference into code, we should follow the format we've done elsewhere and make it configurable via candlepin.conf.

Reply:
@Ceiu 
For now, I have moved ‘hibernate.javax.cache.uri’ property from persistence.xml and JCacheManagerProvider to Candlepin.conf file. However, I feel all related properties should be available in one place. Please suggest whether we should move the other properties related to caching to Candlepin.conf in the PR?

Reply:
I think for now, just the property being added is all we should focus on. Addressing config organization is a fairly large can of worms in itself that should be addressed at some point, but is substantial enough that it probably deserves its own discussion and PR.

How you've handled it seems fine to me, given that we do the same thing in other places. So long as the target file isn't hardcoded, but has a reasonable default in ConfigProperties, we're probably good for the foreseeable future.

That said, the default looks to be the only thing missing at this point. Just need to add that to the bottom blob in ConfigProperties and we should be good.

Reply:
@Ceiu Added default for the 'jpa.config.hibernate.javax.cache.uri' property.